code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -1651,6 +1651,21 @@
 		RND_CATEGORY_MECHFAB_CYBORG_MODULES + RND_SUBCATEGORY_MECHFAB_CYBORG_MODULES_SERVICE
 	)
 
+/datum/design/borg_upgrade_botany
+	name = "Botany Tools"
+	id = "borg_upgrade_botany"
+	build_type = MECHFAB
+	build_path = /obj/item/borg/upgrade/botany_upgrade
+	materials = list(
+		/datum/material/iron = SHEET_MATERIAL_AMOUNT*10,
+		/datum/material/uranium = SHEET_MATERIAL_AMOUNT, // because like.. plants are green :D
```

Comment:
Having uranium as a requirement just because it's green isn't enough justification. Spades are green yet they just require iron & a full sheet of uranium as well is overkill. Remove it

Reply:
its more about balance, like why would cookbook require diamonds? i thought about leaving just metal and glass, but it would be too easy to make imho

Reply:
Titanium makes a bit more sense for tools.

Reply:
> Titanium makes a bit more sense for tools.

so should i add titanium here?